docs/meter-usage-analytics-flow.md

```
@@ -0,0 +1,709 @@
+```
+
+---
+
+## 4. Standard Meter Analytics: `repo.GetDetailedAnalytics()`
+
+**File**: `internal/repository/clickhouse/meter_usage.go:389`
+
+### 4.1 Parse GroupBy: `qb.BuildDetailedGroupByColumns()`
+
+Maps group-by fields to SQL:
+| Field | SQL Column | SQL Alias |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)